test/unit/Validation/Constraint/ValidAtTest.php

```diff
@@ -38,6 +38,7 @@ public function createDependencies(): void
      *
      * @uses \Lcobucci\JWT\Token\DataSet
      * @uses \Lcobucci\JWT\Token\Plain
+     * @uses \Lcobucci\JWT\Token\Signature
```

Comment:
Although the objects are used I don't want to make PHPUnit mark them as covered on the coverage report.

`@covers` and `@uses` are a requirement since I configured PHPUnit to be strict about these annotations.